rules/sonic_bgpcfgd.mk

```diff
@@ -7,7 +7,12 @@ $(SONIC_BGPCFGD)_SRC_PATH = $(SRC_PATH)/sonic-bgpcfgd
 # as part of its unit tests.
 # TODO: Refactor unit tests so that these dependencies are not needed
 
-$(SONIC_BGPCFGD)_DEPENDS += $(SONIC_CONFIG_ENGINE_PY3)
-$(SONIC_BGPCFGD)_DEBS_DEPENDS += $(PYTHON3_SWSSCOMMON)
+$(SONIC_BGPCFGD)_DEPENDS += $(SONIC_CONFIG_ENGINE_PY3) \
+                            $(SONIC_YANG_MGMT_PY3) \
+                            $(SONIC_YANG_MODELS_PY3)
```

Comment:
Do you need to add them explicitly? I guess they can be recursively figure out implicitly.

Reply:
I was under the same impression, I didnt have this in the inital commit but without these changes, the bgpcfg compilations fails.